### PR TITLE
User Interface for Editing Child Information.

### PR DESCRIPTION
**Pull request by laxmankhanal**

### CODE REVIEW

File: Vaccination/AddChildFormViewCell.swift [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import UIKit
+
+class AddChildFormViewCell: UITableViewCell {
+
+    @IBOutlet var firstNameField: UITextField!
+    @IBOutlet var lastNameField: UITextField!
+    @IBOutlet var dateField: UITextField!
+    @IBOutlet var genderLabel: NSLayoutConstraint!
+
+}
```
Comment (sanjaymhj) [inline Vaccination/AddChildFormViewCell.swift]: line at EOF

File: Vaccination/AddChildFormTVC.swift [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+
+        case 2:
+            return "Gender"
+
+        default:
+            return ""
+        }
+    }
+
+
```
Comment (sanjaymhj) [inline Vaccination/AddChildFormTVC.swift]: why extra lines?

File: Vaccination/AddChildFormTV.swift [hunk 3]
```diff
@@ -0,0 +1,5 @@
+import UIKit
+
+class AddChildFormTV: UIViewController {
+
+}
```
Comment (sanjaymhj) [inline Vaccination/AddChildFormTV.swift]: Does AddChildFormTV has to exist if there is no code to be written there?

File: Vaccination/AddChildFormTVC.swift [hunk 4]
```diff
@@ -0,0 +1,106 @@
+import UIKit
+
+class AddChildFormTVC: UITableViewController {
+
+    let numberOfRowsAtSection: [Int] = [1, 1,3]
```
Comment (sanjaymhj) [inline Vaccination/AddChildFormTVC.swift]: make spacings correct

File: Vaccination/AddChildFormTVC.swift [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+    @IBOutlet var nameLabel: UILabel!
+    @IBOutlet var ageLabel: UILabel!
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+    }
+
+    override func didReceiveMemoryWarning() {
+        super.didReceiveMemoryWarning()
+    }
```
Comment (sanjaymhj) [inline Vaccination/AddChildFormTVC.swift]: no need to override function just to call the super of the function

File: Vaccination/AddChildFormViewCell.swift (hunk 1 above)
Comment (laxmankhanal) [inline Vaccination/AddChildFormViewCell.swift]: :ok:

File: Vaccination/AddChildFormTVC.swift (hunk 2 above)
Comment (laxmankhanal) [inline Vaccination/AddChildFormTVC.swift]: :ok:

File: Vaccination/AddChildFormTV.swift (hunk 3 above)
Comment (laxmankhanal) [inline Vaccination/AddChildFormTV.swift]: :ok:

File: Vaccination/AddChildFormTVC.swift (hunk 4 above)
Comment (laxmankhanal) [inline Vaccination/AddChildFormTVC.swift]: :ok:

File: Vaccination/AddChildFormTVC.swift (hunk 5 above)
Comment (laxmankhanal) [inline Vaccination/AddChildFormTVC.swift]: :ok:

**Comment by sanjaymhj** [comment]
To be included in #9

**Pull request closed without merging**